### PR TITLE
:ghost: git ignore vendor folder

### PR DESCRIPTION
**Pull request by 6543**
minor .gitignore change

### CODE REVIEW

**Comment by 6543** [comment]
-> added better description

**Comment by spencerschrock** [comment]
Can you explain the motivation for this change? We don't vendor dependencies in this project, are you trying to locally?

**Comment by 6543** [comment]
Some tooling does work better if you `go mod vendor` a golang prohect first ...

... but as you mention you dont vendor this files in git, so it should be ignored to not interfear

**Comment by 6543** [comment]
@spencerschrock ☝️ ?

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,6 @@ dist/*
 repos.txt
 oldRelease.json
 newRelease.json
+
+# Ignore golang's vendored files
+vendor/
```
Comment (spencerschrock) [inline .gitignore:64]: Let's make this an absolute path, 
```
/vendor/
```

we have a folder with some testdata named "vendor"
```
$ find -name vendor
./checks/raw/testdata/vendor
```

Comment (spencerschrock) [reply .gitignore:64]: And probably our other go.mod folder too then.
```
/tools/vendor/
```

Comment (6543) [inline .gitignore:64]:
```suggestion
/vendor/
/tools/vendor/
```

**Review by spencerschrock — APPROVED** [review]

**Pull request merged**